utils/wallet/provider/uphold/uphold.go

```diff
@@ -115,8 +115,9 @@ func init() {
 		Timeout: time.Second * 60,
 		Transport: middleware.InstrumentRoundTripper(
 			&http.Transport{
-				Proxy:   proxy,
-				DialTLS: pindialer.MakeDialer(upholdCertFingerprint),
+				Proxy:          proxy,
+				DialTLSContext: pindialer.MakeContextDialer(upholdCertFingerprint),
+				DialTLS:        pindialer.MakeDialer(upholdCertFingerprint),
```

Comment:
don't think we need to set both here: 
```
If both are set, DialTLSContext takes priority.
```

Reply:
https://github.com/brave-intl/bat-go/pull/497/commits/4663efa6ccd9af5d3408d5106b480427e700df68 <- fixed here